test/browser/test_sdl2_timer.c

```diff
@@ -10,13 +10,18 @@
 #include <emscripten.h>
 #include <SDL/SDL.h>
 
-Uint32 SDLCALL report_result(Uint32 interval, void *param) {
+Uint32 start_time;
+
+Uint32 SDLCALL timer_callback(Uint32 interval, void *param) {
   static int reported = 0;
-  if (reported) return 0;
+  assert(!reported);
   reported = 1;
-  SDL_Quit();
   int result = *(int *)param;
-  printf("report_result: %p %d\n", param, result);
+  printf("timer_callback: %p %d\n", param, result);
+  assert(result == 5);
```

Comment:
The bad timer with 4 has a timeout of 500 though, shouldn't it happen first?

Reply:
The point of the bad timer is that it should never fire (since it gets cancelled before we return from main).